cmd/grpcurl/grpcurl.go

```diff
@@ -225,6 +231,11 @@ func main() {
 		warn("The -emit-defaults is only used when using json format.")
 	}
 
+	// No protoset or protofiles provided, Use reflection
```

Comment:
At this point, we've already parsed all flags. If there was a `-reflection=false`, which is a condition worth failing with an error message if there were no protos or protosets, we wouldn't know it and would just be overwriting the option.

If you're not sure how to address this (it would require creating a custom implementation of `flag.Value` to track both whether a value was set on the command-line and what that value was), don't worry about it. You can just add a TODO, and I can follow-up with some small cleanup and input validation later.